arch/power/slide_hash_power8.c

```diff
@@ -0,0 +1,58 @@
+ /* Copyright (C) 2019 Matheus Castanho <msc@linux.ibm.com>, IBM
+  * For conditions of distribution and use, see copyright notice in zlib.h
+  */
+
+#include <altivec.h>
+#include "zbuild.h"
+#include "deflate.h"
+
+static inline void slide_hash_power8_loop(deflate_state *s, unsigned n_elems, Pos *table_end)
+{
```

Comment:
Braces should be on the same line as function prototype.

Reply:
Fixed.